docs/src/pages/[platform]/connected-components/authenticator/customization/override-function-calls/example.react.mdx

```diff
@@ -0,0 +1,88 @@
+import { Tabs, Alert } from '@aws-amplify/ui-react';
+import { CURRENT_MAJOR_VERSIONS, PREV_MAJOR_VERSIONS  } from '@/data/frameworks';
+
+export const latestVersion = CURRENT_MAJOR_VERSIONS['react'];
+export const prevVersion = PREV_MAJOR_VERSIONS['react'];
+
+  <Tabs.Container defaultValue={latestVersion}>
+    <Tabs.List>
+      <Tabs.Item value={latestVersion}>ui-react v{latestVersion} (latest)</Tabs.Item>
+      <Tabs.Item value={prevVersion}>ui-react v{prevVersion}</Tabs.Item>
+    </Tabs.List>
+    <Tabs.Panel value={latestVersion}>
+      In `@aws-amplify/ui-react` version {latestVersion}, `Auth` function calls are imported directly as shown below. 
+      ```jsx{2}
+import { Amplify } from 'aws-amplify';
+import { signUp, SignUpInput } from 'aws-amplify/auth';
+import { Authenticator } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+import awsExports from './aws-exports';
+Amplify.configure(awsExports);
+
+export default function AuthenticatorWithEmail() {
+  const services = {
+    async handleSignUp(input: SignUpInput) {
+      // custom username and email
+      const { username, password, options } = input;
+      const customUsername = username.toLowerCase();
+      const customEmail = options?.userAttributes?.email.toLowerCase();
+      return signUp({
+        username: customUsername,
+        password,
+        options: {
+          ...input.options,
+          userAttributes: {
+            ...input.options?.userAttributes,
+            email: customEmail,
+          },
+        },
+      });
+    },
+  };
+  return (
+    <Authenticator services={services} initialState="signUp">
+      {({ signOut }) => <button onClick={signOut}>Sign out</button>}
+    </Authenticator>
+  );
+}
+      ```
+    </Tabs.Panel>
+    <Tabs.Panel value={prevVersion}>
+      <Alert role="none" variation="warning" heading="Sign Up Auto Sign In">
+        If you're overriding `signUp` in `@aws-amplify/ui-react` version {prevVersion} or earlier, you must include the `autoSignIn` key and set `enabled` to true, as shown in the example below.
+      </Alert>
+      In `@aws-amplify/ui-react` version {prevVersion} and earlier, `Auth` function calls are imported from the `Auth` object.
+    ```jsx{1,19}
```

Comment:
something is going on with the code formatting here:
![CleanShot 2024-03-29 at 13 48 24](https://github.com/aws-amplify/amplify-ui/assets/6165315/f9072411-e453-4924-88a8-a40f97b57f7b)

Reply:
Fixed now